Update concepts-overview.md (#8009)
camunda/camunda-docs: docs/components/concepts/concepts-overview.md

## Camunda 8 use cases

With Camunda 8, you can model, execute, and operate end-to-end processes that span microservices, APIs, AI agents, human tasks, legacy systems, IoT devices, and more.

Most real-world automation is distributed. A single business outcome (for example, customer onboarding, claims handling, or order fulfillment) often requires many independently deployed services and external systems. That makes it hard to keep the overall process visible, understand where work is waiting, and recover cleanly when something fails.

Camunda provides a process orchestration layer that coordinates these endpoints without forcing you into a tightly coupled architecture. You define process and decision logic using BPMN and DMN, then run it on a stateful, event-driven workflow engine designed for long-running, high-volume execution. This gives teams a consistent way to trigger work, correlate events, handle retries and compensation, and troubleshoot incidents across the full business process, not just within a single service.

> Many processes also require human input, either as a normal step (for example, review or approval) or as a fallback when automation cannot proceed. Camunda lets you model human tasks alongside automated steps and then assign, track, and escalate work so the process keeps moving. For example, if customer onboarding is blocked waiting for a verification task, the process can route to the right person, enforce due dates, and make the delay visible in operations tooling.

Reviewer: ```suggestion
Many processes also require human input, either as a normal step (for example, review or approval) or as a fallback when automation can't proceed. With Camunda, you can model human tasks alongside automated steps, then assign, track, and escalate work so the process keeps moving. For example, if customer onboarding is blocked waiting for a verification task, the process can route to the right person, enforce due dates, and make the delay visible in operations tooling.
```

- **High availability and fault tolerance** via a pre-configured replication mechanism, ensuring Camunda 8 can recover from machine or software failure with no data loss and minimal downtime. This ensures the system as a whole remains available without requiring manual action.

> - **Audit trail** as all process-relevant events are written to an append-only log, providing an audit trail and a history of the state of a process.

Reviewer 2: Worth adding a sentence here connecting the audit trail to agentic use cases explicitly: This is particularly critical for AI-assisted processes, where every agent decision and action is traceable end-to-end.

- **Operational ease-of-use** because as a SaaS provider, we take care of all operational details.

## What are the Camunda 8 components?

### Modeler

Design fully executable process and decision models that reduce misalignment and handoff friction while giving engineers the freedom they need to build the right solution. Camunda Modeler gives business users an intuitive way to model processes and decisions using the BPMN and DMN standards so their intent is clear, structured, and directly usable by developers. Developers can take the model as-is and build scalable, flexible solutions without worrying about losing alignment with business intent. Available via [web and desktop app](/components/modeler/about-modeler.md).

#### Connectors

Connectors communicate with any system or technology, reducing the time it takes to automate and orchestrate business processes. Use connectors to orchestrate across APIs, microservices, RPA bots, AI/ML tools, enterprise applications, legacy systems, and more. Browse connectors in [Camunda Marketplace](https://marketplace.camunda.com/).

#### AI agents

> Build [enterprise-grade AI agents](/components/agentic-orchestration/ai-agents.md) with guardrails so that they can solve complex problems with autonomy. Camunda implements agentic BPMN that enables teams to model deterministic process logic and dynamic agentic behavior (reasoning loops, memory, prompts, RAG, and human‑in‑the‑loop boundaries) in one unified, executable model.

Reviewer: ```suggestion
Build [enterprise-grade AI agents](/components/agentic-orchestration/ai-agents.md) with guardrails so they can solve complex problems with autonomy. Camunda implements agentic BPMN that enables teams to model deterministic process logic and dynamic agentic behavior, such as reasoning loops, memory, prompts, RAG, and human‑in‑the‑loop boundaries, in one unified, executable model.
```

> Build [enterprise-grade AI agents](/components/agentic-orchestration/ai-agents.md) with guardrails so they can solve complex problems with autonomy. Camunda implements agentic BPMN that enables teams to model deterministic process logic and dynamic agentic behavior, such as reasoning loops, memory, prompts, RAG, and human‑in‑the‑loop boundaries, in one unified, executable model.

Reviewer 2: One suggestion: this section describes what you can model but doesn't connect to how agents interact with external tools at runtime. Worth adding a line noting that agents connect to external systems through Camunda's connector ecosystem, including MCP (Model Context Protocol) support — this ties the modeling capability to actual execution and is a meaningful differentiator for enterprise buyers evaluating Camunda vs. standalone agent frameworks.

### Operate

[Operate](/components/operate/operate-introduction.md) enables teams to monitor running processes, troubleshoot and resolve incidents, and modify and migrate process instances. Trace process flows in real time, investigate failures, modify variables, and resume execution where needed, all within the context of the end-to-end business process.

### Optimize

[Optimize](/components/optimize/what-is-optimize.md) leverages process execution data to continuously [provide actionable insights](/components/optimize/improve-processes-with-optimize.md). Optimize specializes in BPMN-based analysis and can show users exactly what their process model needs for successful execution.

### Console

With [Console](/components/console/introduction-to-console.md), teams can create, configure, manage, and monitor clusters for all environments, from development to production. Console offers control over organizational settings such as user management, roles, as well as insights into usage metrics.

## How does Camunda 8 compare to other solutions?

### Composability

> Camunda's Integrated yet flexible architecture facilitates best-of-breed technology and reuse. Combine Camunda task automation with other tools and custom code for maximum flexibility. Choose SaaS or Self-Managed and use your preferred cloud, Kubernetes, and identity provider.

Reviewer: ```suggestion
Camunda's integrated yet flexible architecture facilitates best-of-breed technology and reuse. Combine Camunda task automation with other tools and custom code for maximum flexibility. Choose SaaS or Self-Managed, and use your preferred cloud, Kubernetes, and identity provider.
```

### Open architecture

Integrate with a variety of applications, systems, and services to scale your architecture. Camunda works with your preferred source control, CI/CD pipelines, and programming languages; while well-documented APIs and SDKs, plus polyglot clients, enable easy integration and customization.

### IT and business collaboration

IT and business stakeholders build and test processes in a shared modeling environment, while reusable processes, decision tables, agents, forms, and connectors make onboarding new teams easy.

### Scalability

> Event streaming avoids database bottlenecks and can scale process throughput infinitely. Camunda's distributed architecture ensures continuity in the case of hardware or network failure and supports replication across global data centers for high availability.

Reviewer: Is "infinitely" the right claim here?

- [Create a Camunda 8 account](/components/console/manage-plan/create-account.md)
- [Migrate from Camunda 7 to Camunda 8](/guides/migrating-from-camunda-7/index.md)
- [Automate a process using BPMN](/components/modeler/bpmn/automating-a-process-using-bpmn.md)

> - [Build your first AI agent](/guides/getting-started-agentic-orchestration.md)

Reviewer: I'd prefer to replace an existing list item instead of adding a new one, as this list of next steps is already quite long. Thoughts @camunda/tech-writers?

Reviewer: We'll just leave it for now, since it sounds like the page will be revisited soon.
